files/image_config/interfaces/init_interfaces

```diff
@@ -0,0 +1,11 @@
+# Initial /etc/network/interface file for first boot 
+# Will be overwritten based on minigraph information by interfaces-config service
+
+# The loopback network interface
+auto lo
+iface lo inet loopback
+#
+# The management network interface
+auto eth0
+iface eth0 inet dhcp
```

Comment:
Not quite sure whether this is the default behavior. Why not override it explicitly to make things clearer though?

---
In reply to: [100564200](https://github.com/Azure/sonic-buildimage/pull/288#discussion_r100564200) [](ancestors = 100564200)